docs/examples/cst/kinesin_closed_loop.rst

```diff
@@ -1,95 +1,109 @@
-Force Clamp - Kinesin Attached to a Bead Walking on Microtubule
-===============================================================
+Kinesin Walking on Microtubule with Force Clamp
+===============================================
 
 .. only:: html
 
     :nbexport:`Download this page as a Jupyter notebook <self>`
 
 
-In this assay we had microtubules on the surface. We trapped beads with Kinesin (molecular motor) and had ATP inside the assay. As we lowered the kinesin-coated beads on top of a microtubule, it attached to it and started stepping on the microtubule. Kinesins were pulling the bead out of the center of the trap and thus increasing the force on the bead.
+In this assay we had microtubules on the surface.
+We trapped beads with Kinesin (molecular motor) and had ATP inside the assay.
+As we lowered the kinesin-coated beads on top of a microtubule, it attached to it and started stepping on the microtubule.
+Kinesins were pulling the bead out of the center of the trap and thus increasing the force on the bead.
 
-At a certain (set) force, we turn the force clamp on and the nanostage moves towards the motion of the bead. Now the force stays constant, and we get an idea of how the motor steps by looking at the motion of the nanostage.
+At a certain (set) force, we turn the force clamp on and the nanostage moves towards the motion of the bead.
+Here the force stays constant, and we get an idea of how the motor steps by looking at the motion of the nanostage.
 
-With the IRM, you can see unlabeled microtubules and the kinesin-coated bead on top of one of them.
+With IRM, you can see unlabeled microtubules and the kinesin-coated bead on top of one of them.
 
 .. image:: kinesin_open_loop_on_MT.png
+  :nbattach:
 
-Open the file::
+Download the files with :func:`~lumicks.pylake.download_from_doi()`::
 
-    file = lk.File("20190215-142512 Marker force clamp.h5")
+    lk.download_from_doi("10.5281/zenodo.12666579", "data")
+
+Open the :class:`~lumicks.pylake.File`::
+
+    file = lk.File("data/stepping_closed_loop.h5")
 
 Load the data::
 
     # Force in the y direction (pN)
-    forcey = file['Force HF']['Force 1y']['6s':'8.5s']
+    force1y = file["Force HF"]["Force 1y"]["6s":"8.5s"]
+
+Load and calibrate the nanostage signal::
+
     # Nanostage position in the y direction (V)
-    nanoy = file['Diagnostics']['Nano Y']['6s':'8.5s']
+    nano_y = file["Diagnostics"]["Nano Y"]["6s":"8.5s"]
 
-Downsample the data::
+    # this is determined for each nanostage and it has 3 different conversion factors for the 3 directions (x,y,z)
+    nano_calibration_factor = 50000 / (1.849-0.04933)  # nm/V
+    nano_y = nano_y * nano_calibration_factor - 2000
```

Comment:
I agree. They don't need to do this anymore (since we calibrate it in the factory for them nowadays), hence the note below.

This data is very old (and I'd much rather replace it with some newer data at some point), but for now, I wanted to get the notebook at least runnable.